museum/sampleRecords/itemData.destination.json

```diff
@@ -0,0 +1,362 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "object_id":"1",
+      "id":"1",
+      "idno":"A1988.135",
+      "display_label":"STRAWBERRY HILL FARM, ALBANY",
+      "ca_objects.Description":"Harper, Mary Alison. Gilt-painted wood frame, unglazed oil of Strawbwrry Hill Farmhouse, Albany, with grey roof. 3 sections: front: 1 storey: 2x6 paned windows, joining building with door &amp; bell LHS, 2-storey 1x9 paned window &amp; chimney. Post\/rail fence. LHS stump; roof height, gateway,no gate. Signed LR.",
+      "ca_objects.MakersMarks":"Harper, Mary Alison",
+      "ca_objects.ItemDates":"1905-",
+      "ca_objects.PhysicalCondition":"2",
+      "ca_objects.Materials":"Oil on hardboard",
+      "ca_objects.Dimensions.DimensionsValue":"H:58.4cms W:81.8cms",
+      "classificationHierarchy":"ILLUSTRATIONS;Oil;building"
+    },
+    {
+      "object_id":"2",
+      "id":"2",
+      "idno":"A1998.16",
+      "display_label":"GALLOP HOUSE",
+      "ca_objects.Description":"Harper, Mary Alison. Unframed, unglazed oil painting of Gallop House overlooking Swan River. Signed: LRC.  Painted c1974.",
+      "ca_objects.MakersMarks":"Harper, Mary Alison",
+      "ca_objects.ItemDates":"c1974",
+      "ca_objects.PhysicalCondition":"2",
+      "ca_objects.Materials":"Oil on Masonite",
+      "ca_objects.Dimensions.DimensionsValue":"H: 45.4cms W: 55.5cms.  Unframed.",
+      "classificationHierarchy":"ILLUSTRATIONS;Oil;building"
+    },
+    {
+      "object_id":"3",
+      "id":"3",
+      "idno":"A1964.26a",
+      "display_label":"TOWN HALL PERTH, ETCHING",
+      "ca_objects.Description":"Brown wood with gilt line frame, glazed, engraving\/ etching of Perth Town Hall clock tower &amp; people on Hay Street corner, painted c1916. Signed LL.",
+      "ca_objects.MakersMarks":"Van Raalte, Henri Benedictus (HenrI)",
+      "ca_objects.ItemDates":"1916",
+      "ca_objects.EarliestYear":"1916",
+      "ca_objects.LatestYear":"1916",
+      "ca_objects.HistoricalDetails":"Henri Van Raalte 1881-1929.   Refer notes D. Erickson's Audit 2014",
+      "ca_objects.PhysicalCondition":"2",
+      "ca_objects.Materials":"Etching on paper",
+      "ca_objects.Dimensions.DimensionsValue":"H: 39.3cms W: 21cms. Frame:",
+      "classificationHierarchy":"Root node for Classification;ILLUSTRATIONS;Etching;building"
```

Comment:
ok this looks like it's working now. I ended up using `<unit relativeTo='ca_list_items'><unit relativeTo='ca_list_items.hierarchy'><unit>^idno (^item_id)</unit></unit></unit`